### ISSUE TITLE
Add a virtual navigation bar 添加虚拟导航栏

### ISSUE THREAD

**Issue by YanYiGe2023** [Genymobile/scrcpy#5299]
操作全屏幕手势的手机很难进行返回、多任务等操作
Phones that operate full-screen gestures have a hard time doing things like going back, multitasking, and so on
![屏幕截图 2024-09-16 212545](https://github.com/user-attachments/assets/ad706bcd-f9f2-49fd-8ba6-2a6a00db5dc7)
因为我很喜欢在需要同时使用手机和电脑进行协助时使用scrcpy
Because I love using scrcpy when I need to use my phone and computer for assistance at the same time
但是每当我想返回时总是会调整窗口大小（我的电脑有触摸屏）
But the window is always resized whenever I want to go back (my computer has a touch screen)
![屏幕截图 2024-09-16 214745](https://github.com/user-attachments/assets/cafe6da8-6894-4f7d-901c-32764aea4001)
有什么办法可以添加虚拟导航栏或者锁定窗口大小吗？
Is there any way to add a virtual navigation bar or lock the window size?

**Comment by rom1v** [comment]
> Is there any way to add a virtual navigation bar

No, but there are shortcuts: https://github.com/Genymobile/scrcpy/issues/7#issuecomment-371772919

> But the window is always resized whenever I want to go back

Not sure to understand. The window is resized when the device resolution changes (due to rotation or something). Is it your case?

**Comment by YanYiGe2023** [comment]
> > 有什么方法可以添加虚拟导航栏
> 
> 否，但有快捷方式：[#7 （评论）](https://github.com/Genymobile/scrcpy/issues/7#issuecomment-371772919)
> 
> > 但是每当我想返回时，窗口总是会调整大小
> 
> 不确定是否理解。当设备分辨率发生变化时（由于旋转或其他原因），窗口的大小会调整。是你的情况吗？
> 
> 编辑：哦，好的，你做一个手势返回，然后你调整窗口的大小，对吗？

我说的不太严谨，当我进行全屏幕手势返回时，Windows误以为我想调整窗口尺寸，于是scrcpy窗口尺寸被调整并出现黑边
I'm not too strict, when I go back with a full-screen gesture, Windows mistakenly thinks I want to resize the window, so the scrcpy window size is resized and black bars appear

**Comment by rom1v** [comment]
OK, then use shortcuts for BACK :wink: (and please write in English)
